using/unix.po

```diff
@@ -144,94 +172,124 @@ msgid ""
 "${exec_prefix}``) are installation-dependent and should be interpreted as "
 "for GNU software; they may be the same."
 msgstr ""
+"Ceux-ci sont sujets à des différences en fonction des conventions locales "
+"d'installation ; :envvar:`prefix` (``${prefix}``) et :envvar:`exec_prefix` "
+"(``${exec_prefix}``) sont dépendants de l’installation et doivent être "
+"interprétés **as for GNU software** ; ils peuvent être égaux."
```

Comment:
Pourquoi ne pas traduire `as for GNU software` ?

Reply:
Parce que je ne comprends pas le sens de la phrase :thinking:

Reply:
> comme pour des logiciels GNU

Il parle de noms de variables, qui chacunes ont un sens, et ce sens est le même que dans les soft GNU classiques.

Reply:
```suggestion
"interprétés comme pour des logiciels GNU ; ils peuvent être égaux."
```